src/styles/index.ts

```diff
@@ -1630,6 +1635,8 @@ const styles = (theme: ThemeColors) =>
             justifyContent: 'space-between',
             borderRightWidth: 1,
             borderColor: theme.border,
+            marginTop: 0,
```

Comment:
Same, can you share a before/after with these changes? I don't quite know what they are doing.